packages/primitives/src/RadioGroup.tsx

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { forwardRef } from "react";
+import { radioGroupAnatomy } from "@ark-ui/anatomy";
+import { RadioGroup } from "@ark-ui/react";
+import { sva } from "@ndla/styled-system/css";
+import { createStyleContext } from "./createStyleContext";
+import { useFormControl } from "./FormControl";
+import { Label } from "./Label";
+import { Text, TextProps } from "./Text";
+
+const radioGroupRecipe = sva({
+  slots: radioGroupAnatomy.keys(),
+  base: {
+    root: {
+      display: "flex",
+      flexWrap: "wrap",
+      position: "relative",
+      gap: "small",
+      _vertical: {
+        flexDirection: "column",
+      },
+      _horizontal: {
+        flexDirection: "row",
+      },
+    },
+    itemControl: {
+      background: "surface.default",
+      borderColor: "stroke.subtle",
+      borderRadius: "full",
+      borderWidth: "2px",
+      width: "medium",
+      height: "medium",
+      outlineStyle: "solid",
+      outlineWidth: "4px",
+      outlineOffset: "-6px",
+      outlineColor: "surface.default",
+      transitionDuration: "normal",
+      transitionProperty: "background, border-color, box-shadow",
+      transitionTimingFunction: "default",
+      _hover: {
+        boxShadow: "0 0 0 4px var(--shadow-color)",
+        boxShadowColor: "surface.actionSubtle.hover.strong",
+      },
+      _checked: {
+        background: "stroke.default",
+        borderColor: "stroke.default",
+      },
+      _disabled: {
+        borderColor: "stroke.disabled",
+        _hover: {
+          borderColor: "stroke.disabled",
+          boxShadow: "none",
+        },
+      },
+    },
+    item: {
+      display: "flex",
+      alignItems: "center",
+      cursor: "pointer",
+      gap: "xxsmall",
+      _disabled: {
+        cursor: "not-allowed",
+      },
+      _focus: {
+        outline: "2px solid",
```

Comment:
Her og synes eg designet har litt tjukkere ring rundt valgt radioknapp.
```suggestion
        outline: "3px solid",
```